### DISCUSSION TITLE
How do you achive this effect?

### DISCUSSION THREAD

**Discussion by jasonhw0808** [keiwando/toonsquid#693] [Q&A]
https://github.com/user-attachments/assets/b6335339-5754-4946-9acc-aa324451cbc9
…

I opened another persons animation, but i was very confused on how to achive this 3D effect. Does anyone know how? It uses rotation.

**Comment by Cat-Butler** [comment]
Probably a variation of this technique, with a hidden hierarchy parent. Probably best to ask the creator directly.
https://www.youtube.com/watch?v=aedLelj_HjU

**Comment by 8-8Sophie8-8** [comment]
with keyframes. Rotating, control points  and scaling Keyframes. ( if you want to achieve exactly the same thing as the video you have to use your brain to study the animation pattern to be able to create the exact thing.  But if you don’t want to recreate the exact same thing you can achieve this type of effect by using these 3 (rotating, control points, and scaling  on a rectangle). You can do lots of fun things with keyframes , if you don’t have much knowledge about animation considering mastering the use of Keyframes first within Toonsquid. They make animation simple.
